roles/grafana/templates/dashboards.yaml.j2

```
@@ -0,0 +1,9 @@
+apiVersion: 1
+
+providers:
+  - name: 'Dashboards'
+    updateIntervalSeconds: 10
+    allowUiUpdates: false
```

since the documentation is not very clear, I'd add some explicit values here
```suggestion
    type: file
    disableDeletion: false
    updateIntervalSeconds: 10
    allowUiUpdates: false
```